assets/js/app/DataLayer.js

```diff
@@ -112,12 +112,13 @@ LocusZoom.PositionsDataLayer = function(){
 
     LocusZoom.DataLayer.apply(this, arguments);  
     this.id = "positions";
-    this.fields = ["id","position","pvalue","refAllele","ld:state"];
+    this.xfield = "position";
+    this.yfield = "pvalue|neglog10";
+    this.fields = ["id", this.xfield, this.yfield, "refAllele", "ld:state"];
 
     this.postget = function(){
         this.data.map(function(d, i){
             this.data[i].ld = +d["ld:state"];
```

Comment:
I wasn't surprised about the way you were doing the conversion so much as it was happening at all. So it was really just about being defensive and forcing a conversion. Because the API should already be returning numeric values.